lib/enum_state_machine/integrations/active_record.rb

```diff
@@ -455,10 +455,13 @@ def define_static_state_initializer
           # attributes passed into #initialize
           define_helper :class, <<-end_eval, __FILE__, __LINE__ + 1
             def column_defaults(*) #:nodoc:
-              result = super
+              super
```

Comment:
Stupid question, but what does this do if you're discarding the result?

Reply:
For all the Rails versions I've referenced in https://github.com/HornsAndHooves/enum_state_machine/pull/9#discussion_r793771999 the return value is `_default_attributes.dup.to_hash`. However, depending on the Rails version, there could be some additional logic before that value is returned — like in Rails 6.1.4.4, [`load_schema`](https://github.com/rails/rails/blob/v6.1.4.4/activerecord/lib/active_record/model_schema.rb#L455) is run in that method.